Track Your Reps/Views/RecentEventsVC.swift

```diff
@@ -0,0 +1,42 @@
+///
```

Comment:
you can just make a UIView, pin its constraints like any other component, then in that separate file / view go forward as normal. so you just want another UIView, not another viewcontroller